monitoring/README.md

```diff
@@ -100,9 +116,16 @@ Any line whose first character is `#` is treated as a comment and ignored.
 
 #### Specifying the Default Grafana Password 
 
-You can set the `GRAFANA_ADMIN_PASSWORD` environment variable to specify the default password for Grafana. If you do not specify a default password, one is randomly generated and displayed during deployment.
+Specify the `GRAFANA_ADMIN_PASSWORD` environment variable in
+`$USER_DIR/monitoring/user.env`to specify the default password for Grafana. If you do not specify a default password, one is randomly generated and displayed during deployment.
 
-### Modify user-values-*.yaml to Change Helm Chart Values
+#### Using Ingress for Cloud Providers
+
+If you are using a cloud provider, you must use ingress, rather than
+NodePorts. Use the samples in the [samples/ingress](/samples/ingress)
+area of this repository to set up either host-based or path-based ingress.
```

Comment:
Please add this section to logging/README.md as well.

Reply:
Section added